libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -22,18 +22,23 @@
 
 #include <AP_Param/AP_Param.h>
 #include <AP_HAL/Util.h>
+#include <AP_HAL/AP_HAL.h>
 #include <AP_Math/AP_Math.h>
 #include "PerfInfo.h"       // loop perf monitoring
 
-#define AP_SCHEDULER_NAME_INITIALIZER(_name) .name = #_name,
+#if BOARD_FLASH_SIZE <= 1024
```

Comment:
Why not `HAL_MINIMIZE_FEATURES`?

Reply:
Most of the 1Mb boards no longer have HAL_MINIMIZE_FEATURES - I am happy to do it this way, but it will increase the flash usage on most boards. Maybe I will do this anyway - seems generally useful